### PR TITLE
Fixed some side bar items take time to load

### PR DESCRIPTION
**Pull request by ghulamhafeez**
### Implementation

Fixed funds & attendance report tabs take time to load in side bar

### CODE REVIEW

File: src/app/layout/components/Sidebar.tsx [hunk 1]
```diff
@@ -71,22 +71,26 @@ export const Sidebar = ({ isSidebarOpen, toggleSidebar }: Props) => {
...
-          {
-            name: "Funds",
-            route: Routes.Fund,
-            icon: <HandCoinsIcon />,
-          },
-        ]
-      : []),
   ];
 
+  const inchargeRoute: Route[] = [
```
Comment (Monfernape) [inline src/app/layout/components/Sidebar.tsx:76]: This should be plural.

File: src/app/layout/components/Sidebar.tsx [hunk 2]
```diff
@@ -71,22 +71,26 @@ export const Sidebar = ({ isSidebarOpen, toggleSidebar }: Props) => {
...
+      icon: <ChartNoAxesCombinedIcon />,
+    },
+    {
+      name: "Funds",
+      route: Routes.Fund,
+      icon: <HandCoinsIcon />,
+    },
+  ];
+
+  const routes = useMemo(() => {
```
Comment (Monfernape) [inline src/app/layout/components/Sidebar.tsx:90]: Nice

**Review by Monfernape — APPROVED** [review]
Can you explain why it was taking time to load? Let's try to put these details in description.

File: src/app/layout/components/Sidebar.tsx (hunk 2 above)
Comment (ghulamhafeez) [reply src/app/layout/components/Sidebar.tsx:90]: Thanks

File: src/app/layout/components/Sidebar.tsx (hunk 1 above)
Comment (ghulamhafeez) [reply src/app/layout/components/Sidebar.tsx]: Fixed

**Review by muhammad-bilal-ali-khokhar — APPROVED** [review]

**Pull request merged**